src/timedisc/timedisc.jl

```diff
@@ -156,6 +156,167 @@ function Base.resize!(integrator::SimpleIntegrator2N, new_size)
 end
 
 
+struct HypDiffN3Erk523Sstar
+  gamma1::SVector{5, Float64}
+  gamma2::SVector{5, Float64}
+  gamma3::SVector{5, Float64}
+  beta::SVector{5, Float64}
+  delta::SVector{5, Float64}
+  c::SVector{5, Float64}
+
+  function HypDiffN3Erk523Sstar()
+    gamma1 = @SVector [0.0000000000000000E+00, 5.2656474556752575E-01, 1.0385212774098265E+00, 3.6859755007388034E-01, -6.3350615190506088E-01]
+    gamma2 = @SVector [1.0000000000000000E+00, 4.1892580153419307E-01, -2.7595818152587825E-02, 9.1271323651988631E-02, 6.8495995159465062E-01]
+    gamma3 = @SVector [0.0000000000000000E+00, 0.0000000000000000E+00, 0.0000000000000000E+00, 4.1301005663300466E-01, -5.4537881202277507E-03]
+    beta   = @SVector [4.5158640252832094E-01, 7.5974836561844006E-01, 3.7561630338850771E-01, 2.9356700007428856E-02, 2.5205285143494666E-01]
+    delta  = @SVector [1.0000000000000000E+00, 1.3011720142005145E-01, 2.6579275844515687E-01, 9.9687218193685878E-01, 0.0000000000000000E+00]
+    c      = @SVector [0.0000000000000000E+00, 4.5158640252832094E-01, 1.0221535725056414E+00, 1.4280257701954349E+00, 7.1581334196229851E-01]
+
+    new(gamma1, gamma2, gamma3, beta, delta, c)
+  end
+end
+
+mutable struct SimpleIntegrator3SstarOptions{Callback}
+  callback::Callback # callbacks; used in Trixi
+  adaptive::Bool # whether the algorithm is adaptive; ignored
+  dtmax::Float64 # ignored
+  tstops::Vector{Float64} # tstops from https://diffeq.sciml.ai/v6.8/basics/common_solver_opts/#Output-Control-1; ignored
+end
+
+function SimpleIntegrator3SstarOptions(callback, tspan; kwargs...)
+  SimpleIntegrator2NOptions{typeof(callback)}(
+    callback, false, Inf, [last(tspan)])
+end
+
+mutable struct SimpleIntegrator3Sstar{RealT<:Real, uType, Params, Sol, Alg, SimpleIntegrator3SstarOptions}
+  u::uType #
+  du::uType
+  u_tmp::uType
+  u_tmp2::uType
+  u_tmp3::uType
```

Comment:
```suggestion
  u_tmp1::uType
  u_tmp2::uType
```
You don't use `u_tmp` below, do you? In that case, you could also change `u_tmp2` to `u_tmp1` and `u_tmp3` to `u_tmp2` below everywhere.

Reply:
Good catch! Renamed as proposed in 5af2593.